torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -406,6 +738,32 @@ def aot_dispatch_autograd(
                 + num_tokens  # See Note [Side-Effectful Tokens in AOTAutograd]
             )
             fake_mode = detect_fake_mode()
+            # TODO(anijain2305) - this changes fx_g signature. But somehow it still
```

Comment:
Just to confirm: by `fx_g` here are you referring to the outer joint graph? (not the inner `invoke_subgraph` joint graphs).

I don't think `run_joint_graph_passes_on_hops()` is changing the inputs/output of the outer `fx_g`? It looks like it's just replacing each `call_function(invoke_subgraph, subgraph, *args)` with a new `call_function(invoke_subgraph, new_subgraph, *new_args)`

Reply:
Sorry, leftover from my prototype copy-paste :| Removing